src/Client/app/(admin)/exams/review/page.tsx

```
@@ -0,0 +1,554 @@
+                                            <h4 className="font-semibold mb-2"> Result</h4>
+                                            <div className="flex items-center gap-5">
+                                                <div className="flex items-center gap-3">
+                                                    <span>Points</span>
+                                                    <input
+                                                        type="number"
+                                                        className="w-16"
+                                                        value={submission.score}
+                                                        onChange={(e) =>
+                                                            updateProblemSubmission(
+                                                                submission.questionId,
+                                                                { score: parseInt( e.target.value )} ) }
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)